helix-core/Cargo.toml

```diff
@@ -17,7 +17,7 @@ helix-syntax = { path = "../helix-syntax" }
 ropey = "1.2"
 smallvec = "1.4"
 tendril = "0.4.2"
-unicode-segmentation = "1.6"
+unicode-segmentation = { git = "https://github.com/unicode-rs/unicode-segmentation.git" }
```

Comment:
I'd prefer to use the crates.io hosted dependency, can't we just wait for `unicode-segmentation` to be updated?

Reply:
I think we can ping the maintainer to get an update publish?

Reply:
Yes, we can wait.

Reply:
Okay, revert this back and I think it's good to merge :)

Reply:
If I revert then it can't be built:
```
$ cargo check
...
    Checking helix-core v0.0.10 (/home/begin/Templates/rust/helix/helix-core)
error[E0277]: `GraphemeCursor` doesn't implement `Debug`
   --> helix-core/src/graphemes.rs:156:5
    |
156 |     cursor: GraphemeCursor,
    |     ^^^^^^^^^^^^^^^^^^^^^^ `GraphemeCursor` cannot be formatted using `{:?}` because it doesn't implement `Debug`
    |
    = help: the trait `Debug` is not implemented for `GraphemeCursor`
    = note: required because of the requirements on the impl of `Debug` for `&GraphemeCursor`
    = note: required for the cast to the object type `dyn Debug`
    = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error

For more information about this error, try `rustc --explain E0277`.
error: could not compile `helix-core`

To learn more, run the command again with --verbose.
```

Reply:
Just drop `Debug` from `RopeGraphemes` for the time being.